corebuild/integration/ILLink.Tasks/Utils.cs

```diff
@@ -12,4 +12,15 @@ public static bool IsManagedAssembly (string fileName)
 			return false;
 		}
 	}
-}
\ No newline at end of file
+
+	public static bool IsReadyToRunAssembly (string fileName)
+	{
+		try {
+			ModuleDefinition module = ModuleDefinition.ReadModule (fileName);
+			return (module.Attributes & ModuleAttributes.ILOnly) == 0 &&
+				(module.Attributes & (ModuleAttributes) 0x04) != 0;
```

Comment:
Can you use a descriptive name for this constant? Should we update ModuleDefinition enum in Cecil? Also, this logic will need to be generalized to handle re-crossgen-ing of System.Private.CoreLib, which is not ReadyToRun compiled.

Reply:
We discussed making cecil aware of these flags in https://github.com/jbevain/cecil/issues/390, but Jb didn't think cecil was the right place. Let's make the generalization for SPC along with the other linker changes for linking SPC.

Reply:
On second thought, maybe Jb would be open to adding that enum value as long as we have the logic to handle it. I'll make a PR in cecil. I'm also going to factor this so that the logic lives in one place (right now it's duplicated in https://github.com/mono/linker/blob/master/linker/Mono.Linker.Steps/OutputStep.cs#L90).

Reply:
Cecil pr: https://github.com/jbevain/cecil/pull/475